static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,667 @@
+import {Organization, ReleaseProject} from 'app/types';
+import {TableData, TableDataRow} from 'app/utils/discover/discoverQuery';
+import EventView, {MetaType} from 'app/utils/discover/eventView';
+import {getFieldRenderer} from 'app/utils/discover/fieldRenderers';
+import {getAggregateAlias} from 'app/utils/discover/fields';
+import {getDuration} from 'app/utils/formatters';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+import {GridCell, GridCellNumber} from 'app/views/performance/styles';
+import {TrendsDataEvents} from 'app/views/performance/trends/types';
+import {DiscoverField} from 'app/views/releases/detail/overview/index';
+import {performanceCardLabels} from 'app/views/releases/detail/utils';
+
+const FRONTEND_PLATFORMS: string[] = [...frontend, ...mobile];
+const BACKEND_PLATFORMS: string[] = [...backend, ...serverless];
+
+type ReleaseTableDataRow = {
+  [key: string]: React.ReactText;
+};
+
+export type ReleaseTableData = {
+  data?: Array<ReleaseTableDataRow>;
+  meta?: MetaType;
+};
+
+type Props = {
+  eventView: EventView;
+  releaseEventView: EventView;
+  organization: Organization;
+  project: ReleaseProject;
+  location: Location;
+  isLoading: boolean;
+  tableData: TableData | TrendsDataEvents | null;
+  releaseTableData: ReleaseTableData | null;
+  columnOrder: TableColumn<React.ReactText>[];
+  generateLink?: Record<
+    string,
+    (
+      organization: Organization,
+      tableRow: TableDataRow,
+      query: Query
+    ) => LocationDescriptor
+  >;
+};
+
+class PerformanceCardTable extends React.PureComponent<Props> {
+  renderRow(
+    row: TableDataRow | object,
+    rowIndex: number,
+    columnOrder: TableColumn<React.ReactText>[],
+    tableMeta: MetaType
+  ): React.ReactNode[] {
+    const {organization, location} = this.props;
+
+    const resultsRow = columnOrder.map((column, index) => {
+      const field = String(column.key);
+      // TODO add a better abstraction for this in fieldRenderers.
+      const fieldName = getAggregateAlias(field);
+      const fieldType = tableMeta[fieldName];
+
+      const fieldRenderer = getFieldRenderer(field, tableMeta);
+      let rendered = fieldRenderer(row, {organization, location});
+
+      const isNumeric = ['integer', 'number', 'duration'].includes(fieldType);
+      const key = `${rowIndex}:${column.key}:${index}`;
+      rendered = isNumeric ? (
+        <GridCellNumber>{rendered}</GridCellNumber>
+      ) : (
+        <GridCell id="border">{rendered}</GridCell>
+      );
+
+      return <BodyCellContainer key={key}>{rendered}</BodyCellContainer>;
+    });
+
+    return resultsRow;
+  }
+
+  renderChangeRow(row: object) {
+    const cells: any[] = [];
+
+    Object.values(row).forEach(r => cells.push(r));
+
+    const rendered = (
+      <BodyCellContainer id="changeBody">
+        {cells.map(cell => (
+          <GridCellNumber key={cell} id={!cell ? 'changeBorder' : 'changeNoBorder'}>
+            <StyledIconArrow
```

Comment:
Should the icon be after the number? And only rendered if there's a non-zero diff.